### ISSUE TITLE
"Non-retriable status code 403 while downloading" but an altered URL works

### ISSUE THREAD

**Issue by Ectomind1990** [RipMeApp/ripme#384]
* Ripme version: 1.7.11
* Java version: 1.8
* Exact URL you were trying to rip when the problem occurred: (SFW) https://www.instagram.com/kutovakika/
* Please include any additional information about how to reproduce the problem:

It happens on other instagram accounts also

## Expected Behavior

Download photos and videos from all 495 posts on that account

## Actual Behavior

The 1st 24 downloads are green. But then I get some errors like this
```
Downloading https://scontent-ams3-1.cdninstagram.com/vp/f7ec41a18e8d7541b2863227a0efeadf/5AF7FF03/t51.2885-15/sh0.08/e35/23969950_168007863943538_6966510011505704960_n.jpg
https://scontent-ams3-1.cdninstagram.com/vp/f7ec41a18e8d7541b2863227a0efeadf/5AF7FF03/t51.2885-15/sh0.08/e35/23969950_168007863943538_6966510011505704960_n.jpg : Non-retriable status code 403 while downloading https://scontent-ams3-1.cdninstagram.com/vp/f7ec41a18e8d7541b2863227a0efeadf/5AF7FF03/t51.2885-15/sh0.08/e35/23969950_168007863943538_6966510011505704960_n.jpg
Downloading https://scontent-ams3-1.cdninstagram.com/vp/d9d12f374929b27b7a49b057c88197ae/5AE9D4F8/t51.2885-15/sh0.08/e35/24125316_1290432391060987_7882780828147646464_n.jpg
https://scontent-ams3-1.cdninstagram.com/vp/d9d12f374929b27b7a49b057c88197ae/5AE9D4F8/t51.2885-15/sh0.08/e35/24125316_1290432391060987_7882780828147646464_n.jpg : Non-retriable status code 403 while downloading https://scontent-ams3-1.cdninstagram.com/vp/d9d12f374929b27b7a49b057c88197ae/5AE9D4F8/t51.2885-15/sh0.08/e35/24125316_1290432391060987_7882780828147646464_n.jpg
```

If you remove vp/f7ec41a18e8d7541b2863227a0efeadf/5AF7FF03/ from the 1st failed URL and try it manually it works.
If you remove vp/d9d12f374929b27b7a49b057c88197ae/5AE9D4F8/ from the 2nd failed URL and try it manually it works.

Been seeing this for about 1-2 days. So have instagram just made a change to how their URL's are working?

**Comment by cyian-1756** [comment]
> So have instagram just made a change to how their URL's are working?

It looks like that

**Comment by cyian-1756** [comment]
@Ectomind1990 This should be fixed in 1.7.12
